src/WebJobs.Script/Config/ExtensionBundleConfigurationHelper.cs

```diff
@@ -83,9 +82,8 @@ private void ConfigureProbingPaths(ExtensionBundleOptions options)
                 && !string.IsNullOrEmpty(homeDirectory))
             {
                 string linuxDefaultPath = Path.Combine(Path.GetPathRoot(homeDirectory), ScriptConstants.DefaultExtensionBundleDirectory, options.Id);
-                string deploymentPackageBundlePath = Path.Combine(
-                    homeDirectory,
-                    "site", "wwwroot", ".azureFunctions", ScriptConstants.ExtensionBundleDirectory, options.Id);
+                string deploymentPackageBundlePath = Path.Combine(homeDirectory, "site", "wwwroot", ".azureFunctions",
```

Comment:
Can you use `AzureFunctionsSystemDirectoryName` here instead of `".azureFunctions"`?

Reply:
👍🏼 
Will update. This was existing code that was moved around.